substrate/frame/revive/rpc/src/lib.rs

```diff
@@ -251,10 +251,13 @@ impl EthRpcServer for EthRpcServerImpl {
 		block: BlockNumberOrTag,
 		hydrated_transactions: bool,
 	) -> RpcResult<Option<Block>> {
+		log::debug!(target: LOG_TARGET, "get_block_by_number({block:?})");
 		let Some(block) = self.client.block_by_number_or_tag(&block).await? else {
+			log::debug!(target: LOG_TARGET, "block {block:?} not found");
 			return Ok(None);
 		};
-		let block = self.client.evm_block(block, hydrated_transactions).await?;
+		log::debug!(target: LOG_TARGET, "Building evm block");
+		let block = self.client.evm_block(block, hydrated_transactions).await;
```

Comment:
Shouldn't that be trace. I expect it to be a lot of spam.

Reply:
yeah sorry was debugging, this I can remove actually